object-store-client-play-26/src/main/scala/uk/gov/hmrc/objectstore/client/play/PlayObjectStoreReads.scala

```diff
@@ -21,15 +21,15 @@ import akka.util.ByteString
 import play.api.http.Status
 import play.api.libs.json._
 import play.api.libs.ws.WSResponse
-import uk.gov.hmrc.objectstore.client.model.http.{ObjectStoreRead, ObjectStoreWrite, ObjectStoreWriteData}
+import uk.gov.hmrc.objectstore.client.model.http.ObjectStoreRead
 import uk.gov.hmrc.objectstore.client.model.objectstore
 import uk.gov.hmrc.objectstore.client.model.objectstore.ObjectListing
 
 import scala.concurrent.{ExecutionContext, Future}
 
-trait ObjectStoreReads {
+trait PlayObjectStoreReads {
 
-  implicit def playFutureObjectStoreRead(implicit ec: ExecutionContext): ObjectStoreRead[Future[WSResponse], Source[ByteString, _], Future] =
+  implicit def future(implicit ec: ExecutionContext): ObjectStoreRead[Future[WSResponse], Source[ByteString, _], Future] =
```

Comment:
Should this be called `akkaSource` (or `futureAkkaSource`)?
I'm thinking we should add the suffix `Reads` to these, and `Writes` to PlayObjectStoreWrites, since they are defined on traits, the location doesn't serve as a namespace - e.g. `import PlayWsHttpClient.akkaSourceWrites`